test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1218,6 +1212,12 @@ class test_ucp_sockaddr_wireup_fail : public test_ucp_sockaddr_wireup {
             ASSERT_LT(ucs_get_time(), deadline);
             progress();
         }
+
+        if (m_num_fail_injections == 0) {
+            EXPECT_EQ(0, m_err_count);
+            UCS_TEST_SKIP_R("failure injection was not done, but endpoint was"
+                            " connected");
```

Comment:
IMO better just print UCS_TEST_MESSAGE and not skip halfway through the test
this also we would test disconnect does not hang

Reply:
done